src/main/java/net/sf/jabref/event/ChangeFieldEvent.java

```diff
@@ -0,0 +1,22 @@
+package net.sf.jabref.event;
+
+
+public class ChangeFieldEvent {
```

Comment:
Vote against it. Adding a field is changing it from nothing to something, similar for removed. Thus I think `FieldChangedEvent` is enough.